pkgs/desktops/gnome/extensions/extensionRenames.nix

```
@@ -23,6 +23,15 @@
   "noannoyance@sindex.com" = "noannoyance";
   "noannoyance@daase.net" = "noannoyance-2";
 
+  "clipboard-indicator@tudmotu.com" = "clipboard-indicator";
+  "clipboard-indicator@Dieg0Js.github.io" = "clipboard-indicator-2";
+
+  "KeepAwake@jepfa.de" = "keep-awake";
+  "awake@vixalien.com" = "keep-awake-2";
```

```suggestion
  "awake@vixalien.com" = null;
```

"[DEPRECATED: Use "Caffeine" instead]"